src/App.js

```diff
@@ -7,8 +7,39 @@ import Footer from "./Components/Footer/Footer";
 import RandomContributors from "./Components/randomcontributor/RandomContributors";
 
 const App = () => {
+  const [showButton, setShowButton] = useState(false);
+  const handleScrollToTop = () => {
+    window.scrollTo({ top: 0, behavior: "smooth" });
+  };
+
+  const scrollToTopBtn = () => {
+    return (
+      <div
+        onClick={handleScrollToTop}
+        className={showButton ? "scroll-to-top-btn" : "hideBtn"}
+      >
+        <p className="scroll-to-top-arrow">^</p>
+      </div>
+    );
+  };
+
+  useEffect(() => {
+    const handleScroll = () => {
+      const currentPosition = window.scrollY;
+      setShowButton((prev) =>
+        currentPosition < 1000 ? (prev = false) : (prev = true)
```

Comment:
Very neat as you have it set for the scroll button to show on the page when the user has scrolled down more than 1000 pixels from the top. I would agree this is a good approach👏🏻